sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnHarness.java

```diff
@@ -369,6 +358,16 @@ private BeamFnApi.ProcessBundleDescriptor loadDescriptor(String id) {
 
       LOG.info("Entering instruction processing loop");
```

Comment:
Should we keep this log after beamFnStatusClient initialization?

Reply:
I don't think that's necessary, the status handling is considered part of the instruction processing now.